__tests__/app.test.js

```diff
@@ -76,3 +86,57 @@ describe('GET/api/articles/:article_id', () => {
   });
 });
 
+describe('GET/api/articles/:article_id/comments', () => {
+  it('GET:200 and should respond with an an array of comments for the given article_id', () => {
+    return request(app)
+      .get('/api/articles/1/comments')
+      .expect(200)
+      .then((response) => {
+        const { comments } = response.body;
+        console.log(comments);
```

Comment:
This test is very thorough! 🏆  Don't forget to clear out old console logs from when you were building the test 🧹

Reply:
done